arrow/translate.py

```diff
@@ -36,12 +36,16 @@ def make_op(key, value):
                 value = 'drs://' + value
             return _make_add_update_op(key, value)
 
-        operations = [make_op(key, value)
+        attributes = [make_op(key, value)
                       for key, value in record['object'].items() if value is not None]
+        relations = [make_op(r['dst_name'],
+                             {'entityType': r['dst_name'], 'entityName': r['dst_id']})
+                     for r in record['relations']]
```

Comment:
nit: for clarity could 'r' be renamed something like 'record_relation' (or a better name just couldn't come up with one off the top of my head)

Reply:
I tend to default to short (single-character) names for functional style code like this when possible. You make a good point in this case. The list comprehension is probably just about on the threshold for length where single-character names are still readable, and we're dealing with both records and relations here so `r` isn't sufficiently unambiguous.

Does `relation` work for you?
```
relations = [make_op(relation['dst_name'],
                     {'entityType': relation['dst_name'], 'entityName': relation['dst_id']})
             for relation in record['relations']]
```

Reply:
'relation' sounds great to me!